HealthMonitoring.SelfHost/HealthMonitoring.SelfHost.csproj

```diff
@@ -239,7 +253,18 @@
       <Name>HealthMonitoring</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
```

Comment:
this also was automatically generated in the solution. There are quite a number of elements within it too